<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VRViewWidget</class>
 <widget class="QWidget" name="VRViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VRViewWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#initPage{
background-color:#1f2020;
}
#mapPage{
background:transparent;
}
#label{
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:rgba(238,238,238,1);
line-height:17px;
}
#arrowButton{
border-image:url(:/image/image/nextArrow.png);
}
#arrowButton:hover{
border-image:url(:/image/image/nextArrowh.png);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="initPage">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="topMargin">
        <number>246</number>
       </property>
       <property name="bottomMargin">
        <number>300</number>
       </property>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>208</width>
             <height>17</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>208</width>
             <height>17</height>
            </size>
           </property>
           <property name="text">
            <string>请先在导览设置中添加地图与全景机</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="arrowButton">
           <property name="minimumSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="MapWgt" name="mapPage"/>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MapWgt</class>
   <extends>QWidget</extends>
   <header>MapWgt.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
